pytorch_lightning/plugins/native_amp.py

```diff
@@ -36,15 +36,24 @@ def training_step(self, fx, args):
             output = fx(*args)
         return output
 
+    @staticmethod
+    def is_loss_invalid(loss):
+        return torch.isnan(loss) or torch.isfinite(loss)
```

Comment:
`torch.isfinite` includes nans. but we need to support 1.3 and above:

```suggestion
        if LooseVersion(torch.__version__) >= "1.6.0":
            return torch.isfinite(loss).logical_not()
        return (
            (loss != loss).any()
            or (loss == float('inf')).any() or (loss == float('-inf')).any()
        )
```